R/mail.R

```diff
@@ -0,0 +1,62 @@
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+
+
+## Creates fake mbox data
+#' @param mlist specific mailing list associated with this mbox data
+#' @param sender information of person sending the mail
+#' @param date First year in the range to be downloaded
+#' @param subject subject of the email
+#' @param content the body of the email
+#' @param randomise_email_case If TRUE, randomizes the email case. (Default = TRUE)
+#' @export
+create_fake_mbox <- function(mlist, sender, date, subject, content, randomise_email_case = TRUE) {
+
+# format the date correctly
+  cdate <- format(as.POSIXct(date, format = "%Y-%m-%dT%H:%M:%S"), "%a, %d %b %Y %H:%M:%S")
```

Comment:
timezone is missing here. This is very important to not be missed, as it is a variable often used on mailing list analysis. Our mail fake data generator should be able to let us choose the timezone information when specifying the date field.